### DISCUSSION TITLE
Time synchronization issues ongoing with podman machine on macOS

### DISCUSSION THREAD

**Discussion by aj-stein-gsa** [containers/podman#24160] [Q&A]
Hello, I am not sure what the current guidance or recommendation is for properly configuring podman on macOS. Current system and version info:

```
me@computer $ sw_vers -productVersion                   
14.6.1 # macOS operating system version
$ brew --version
Homebrew 4.3.24 # brew version
me@computer $ brew info podman
==> podman: stable 5.2.3 (bottled), HEAD
# ... truncated for brevity
me@computer $ podman machine ssh hostnamectl
     Static hostname: (unset)
  Transient hostname: localhost
           Icon name: computer-vm
             Chassis: vm 🖴
          Machine ID: 265f0525f79e4b82a64cc29ba4c1e025
             Boot ID: 3a8278235a204ecf9317afc8917d5a4a
      Virtualization: apple
    Operating System: Fedora CoreOS 40.20240920.2.0
         CPE OS Name: cpe:/o:fedoraproject:fedora:40
      OS Support End: Tue 2025-05-13
OS Support Remaining: 7month 1w 3d
              Kernel: Linux 6.10.10-200.fc40.aarch64
        Architecture: arm64
     Hardware Vendor: Apple Inc.
      Hardware Model: Apple Virtualization Generic Platform
    Firmware Version: 2022.140.5.0.0
       Firmware Date: Sun 2024-06-30
        Firmware Age: 3month 2d
```

I have now had to reset my backing podman VM with a `podman machine reset && podman machine init` a few times due to intermittent system issues. I read up https://github.com/containers/podman-compose/issues/354 and see some related commits and issues in 2023, but it seems to have died down. Is there good guidance on how to make sure time skew does not occur with the underlying VM? I have had skew range from a few minutes to days even after doing some `podman machine stop && podman machine start` and trying to play with timezone and sync settings. One time I swore I changed from UTC to local time too to see what happened and that doesn't necessarily persist. How can I ensure the time sync work as intended these days with `podman`? I notice this because in `podman-compose` and `podman build` signed Debian/Ubuntu release info for `apt` fails because signature are not valid for X days Y hours Z minutes and the like. I can open a new issue, but Google wasn't clarifying for me what should be done, or if this behavior warrants a new bug.

In either, thanks for the excellent Podman project and its various tools. I enjoy them very much!

**Comment by bbc-russellb** [comment]
Same issue here, on a macbook pro M4 laptop. Issue mainly occurs after the laptop has been in sleep mode / suspended. Forced to restart the podman machine to sort it. Not sure if there is an open issue for this?

**Reply by Luap99** [reply]
https://github.com/podman-container-tools/podman/issues/28345

This is supposed to be fixed with the 6.0 build so please test rc1 and let us know if it works properly now
https://github.com/podman-container-tools/podman/releases/tag/v6.0.0-rc1
